etc/tendrl/node-agent/logging.yaml.syslog.sample

```diff
@@ -1,31 +1,17 @@
 version: 1
 disable_existing_loggers: False
-formatters:
-    simple:
-        format: "%(asctime)s - %(filename)s:%(lineno)s - %(funcName)20s() - %(levelname)s - %(message)s"
-        datefmt: "%Y-%m-%dT%H:%M:%S%z"
 
```

Comment:
i have removed format because we cant tell function name and line no and file name from logger , it is passed with message and appended with log message automatically.